go.sum

```diff
@@ -421,6 +421,10 @@ github.com/tarantool/go-prompt v0.2.6-tarantool h1:/dYMRBuM5nE3mleka/mqJWPf8SrJ1
 github.com/tarantool/go-prompt v0.2.6-tarantool/go.mod h1:8enZKIgoGFEQu2XPBK79TguJG2XF3SR4QU2iYI28NSo=
 github.com/tarantool/go-prompt v1.0.0 h1:7lDXh+SBCf4/S3xmDPNOxTA4yfWugD+t82AZOkQviAE=
 github.com/tarantool/go-prompt v1.0.0/go.mod h1:9Vuvi60Bk+3yaXqgYaXNTpLbwPPaaEOeaUgpFW1jqTU=
```

Comment:
Old entries should be deleted in theory. See my patch in the VK Teams message.

Reply:
I did `go mod tidy` for cleaning up old versions.